tests/helpers_tests/test_condition.py

```diff
@@ -24,6 +24,17 @@
 integerCondition = ['num_users', 10, 'custom_attribute', 'exact']
 doubleCondition = ['pi_value', 3.14, 'custom_attribute', 'exact']
 
+eq_semver_condition_list_variation_1 = [['Android', "2.0", 'custom_attribute', 'semver_eq']]
```

Comment:
Give proper names, so anyone seeing testcase, should able to judge what this variable holds.